source/_posts/2016-05-11-deploying-with-capistrano-3.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: "Deploying with capistrano 3"
```

Comment:
@Ikera maybe mention Rails somewhere in title? For example: "Deploying Rails app with Capistrano 3". Also Capistrano should be capitalized.